client/layout/masterbar.jsx

```diff
@@ -1,154 +1,101 @@
 		};
 	},
 
-	getNotificationLinkDomNode: function() {
-		return this.refs.masterbar.refs.notificationLink;
+	clickMySites() {
+		layoutFocus.setNext( 'sidebar' );
 	},
 
-	toggleNotesFrame: function( event ) {
-		if ( event ) {
-			event.preventDefault();
-		}
-
-		this.setState( {
-			showNotes: ! this.state.showNotes
-		}, function() {
-			if ( this.state.showNotes ) {
-				this.setNotesIndicator( 0 );
-			}
-
-			// focus on main window if we just closed the notes panel
-			if ( ! this.state.showNotes ) {
-				this.getNotificationLinkDomNode().blur();
-				window.focus();
-			}
-		}.bind( this ) );
+	clickReader() {
+		layoutFocus.setNext( 'content' );
 	},
 
-	checkToggleNotes: function( event, forceToggle ) {
-		var target = event ? event.target : false,
-			notificationNode = this.getNotificationLinkDomNode();
-
-		// Some clicks should not toggle the notifications frame
-		if ( target === notificationNode || target.parentElement === notificationNode ) {
-			return;
-		}
-
-		if ( this.state.showNotes || forceToggle === true ) {
-			this.toggleNotesFrame();
-		}
-	},
-
-	/**
-	 * Uses the passed number of unseen notifications
-	 * and the locally-stored cache of that value to
-	 * determine what state the notifications indicator
-	 * should be in: on, off, or animate-to-on
-	 *
-	 * @param {Number} currentUnseenCount Number of reported unseen notifications
-	 */
-	setNotesIndicator: function( currentUnseenCount ) {
-		var existingUnseenCount = store.get( 'wpnotes_unseen_count' ),
-			newAnimationState = this.state.animationState;
-
-		// Having no record of previously unseen notes is
-		// functionally equal to having a record of none
-		if ( null === existingUnseenCount ) {
-			existingUnseenCount = 0;
-		}
-
-		if ( 0 === currentUnseenCount ) {
-			// If we don't have new notes at load-time, remove the `-1` "init" status
-			newAnimationState = 0;
-		} else if ( currentUnseenCount > existingUnseenCount ) {
-			// Animate the indicator bubble by swapping CSS classes through the animation state
-			// Note that we could have an animation state of `-1` indicating the initial load
-			newAnimationState = ( 1 - Math.abs( this.state.animationState ) );
-		}
-
-		store.set( 'wpnotes_unseen_count', currentUnseenCount );
-
+	clickNotifications() {
 		this.setState( {
-			newNote: ( currentUnseenCount > 0 ),
-			animationState: newAnimationState
+			showNotifications: ! this.state.showNotifications
 		} );
 	},
 
-	renderMenu: function() {
-		if ( this.props.user ) {
-			return <MasterbarSectionsMenu ref="masterbar" user={ this.props.user } section={ this.props.section } sites={ this.props.sites } toggleNotesFrame={ this.toggleNotesFrame } showNotes={ this.state.showNotes } newNote={ this.state.newNote } animationState={ this.state.animationState } />;
-		}
-		return <MasterbarLoggedOutMenu />;
+	checkIsActive( section ) {
+		return !! ( section === this.props.section && ! this.state.showNotifications );
```

Comment:
the `!! ( ... )` part is not necessary here, since both sides of the `&&` will always be booleans. Also, `isActive()` might be enough for the function name — I'm not sure `check` is adding anything.